src/crypto/x509/pkcs8.go

```diff
@@ -74,6 +76,20 @@ func ParsePKCS8PrivateKey(der []byte) (key interface{}, err error) {
 		}
 		return ed25519.NewKeyFromSeed(curvePrivateKey), nil
 
+	case privKey.Algo.Algorithm.Equal(oidPublicKeyEdDilithium3):
+		if l := len(privKey.Algo.Parameters.FullBytes); l != 0 {
+			return nil, errors.New("x509: invalid Ed25519-Dilithium3 private key parameters")
+		}
+		var packedSk []byte
```

Comment:
I think they name in Golang secret keys 'Private Keys', so maybe is better that name.

Reply:
In APIs I use PrivateKey/PublicKey, but in function bodies having the `sk` and `pk` abbreviations is just so convenient as compared to `pubKey` and `privKey`.

Reply:
Yes, but as you can see on the ed25519 case, the golang people seem to be using (consistently everywhere else) the `privateKey` term as in `var curvePrivateKey []byte`. Might be good to keep that same style, and not diverge from the style of upstram.